libs/blocks/accordion/accordion.js

```diff
@@ -10,7 +11,40 @@ function setSEO(questions) {
   document.head.append(script);
 }
 
-function handleClick(el, dd) {
+function displayMedia(displayArea, el, dd, i, id, expanded) {
+  const theID = el.getAttribute('aria-controls').split('-')[1];
+  [...mediaCollection[theID]].map(
+    (mediaCollectionItem, idx, total) => {
+      mediaCollectionItem.classList.remove('expanded');
+
+      for (let index = 0; index < total.length; index++) {
+        mediaCollectionItem.classList.remove('expanded');
+        const trigger = document.querySelector(`#accordion-${theID}-trigger-${index + 1}`);
+        const content = document.querySelector(`#accordion-${theID}-content-${index + 1}`);
+        trigger.setAttribute('hidden', '');
+        trigger.setAttribute('aria-expanded', 'false');  
+        content.setAttribute('hidden', '');
+        content.setAttribute('aria-expanded', 'false');
+      }
+      el.setAttribute('aria-expanded', 'true');;
+      el.removeAttribute('hidden');
+      dd.setAttribute('aria-expanded', 'true');;
+      dd.removeAttribute('hidden');
+      displayArea.childNodes[i - 1].classList.add('expanded')
+
+      // toggle
+      if (expanded) {
+        el.setAttribute('hidden', '');
+        el.setAttribute('aria-expanded', 'false');  
+        dd.setAttribute('hidden', '');
+        dd.setAttribute('aria-expanded', 'false');
+        displayArea.childNodes[i - 1].classList.remove('expanded')
```

Comment:
missing semicolon + maybe we should rewrite as `displayArea.childNodes[i - 1]?.classList.remove('expanded')` to make sure the element is available.